### DISCUSSION TITLE
✨ [Feature]:  Layout 是否需要更多的功能

### DISCUSSION THREAD

**Discussion by GaoNeng-wWw** [DevCloudFE/vue-devui#1368] [General]
### What problem does this feature solve

最近使用devui来尝试开发的时候发现的一个问题。

## ``<a-side>``
a-side组件感觉缺少了很多东西，比如断点和收缩功能。如果开发者还需要从头开始写响应式，那为什么还要选择我们的库。

## ``<d-aside>``
- collapsed
    - boolean
    - 默认收起
- breakpoint
    - object
    - 断点
- @collapsed
    - Function
    - 收缩时回调
---

**Comment by kagol** [comment]
@GaoNeng-wWw 其实我们缺少一个`Menu`菜单组件，可以增加一个`Menu`组件，该组件提供`收缩`功能，然后用户可以将`Menu`组件和`aside`组件组合使用，就能实现`collapsed`和`@Collapsed`，而且不需要给`aside`增加新的`api`。

**Comment by kagol** [comment]
@GaoNeng-wWw 响应式其实是一个基础能力，我在想是否可以开发一个`useBreakpoint`的`Composable`，然后需要断点的组件可以进行使用，`Vue DevUI`的用户也可以使用。

**Comment by GaoNeng-wWw** [comment]
可以开发一个```useBreakpoint```的指令，让开发者自组选择是否使用断点。Menu组件也十分的有必要
